src/wallet/rpcdump.cpp

```diff
@@ -27,6 +27,10 @@ using namespace std;
 void EnsureWalletIsUnlocked();
 bool EnsureWalletIsAvailable(bool avoidException);
 
+Value _dumpwallet(const Array& params, bool fHelp, bool fDumpZKeys);
+Value _importwallet(const Array& params, bool fHelp, bool fImportZKeys);
```

Comment:
Don't start names with ``_``. These particular names aren't reserved to the C++ implementation, but only just: http://stackoverflow.com/a/12924268/393146